### PR TITLE
Fix sticky diff stats container

### PR DESCRIPTION
**Pull request by CirnoT**
Use pure CSS `position: sticky` and disable Fomantic sticky component

chrome_2020-06-21_13-41-48
chrome_2020-06-21_13-42-02

The border is applied always because there isn't any way to check if element is stuck or not without delegating to JS.

### CODE REVIEW

**Comment by silverwind** [comment]
Not a huge fan of JS-based sticky implementations as CSS `position: sticky` is usually superior if it can be used. Would it be possible here?

**Comment by CirnoT** [comment]
We are using Fomantic sticky element however, so no, it is not.

**Comment by CirnoT** [comment]
A quick test show that

```
  position: sticky;
  top: 0;
```

seem to work. Should we stop using Fomantic's sticky element in that case?

File: web_src/less/_repository.less [hunk 1]
```diff
@@ -1567,6 +1567,14 @@
         background: #ffffff;
         line-height: 30px;
 
+        &.sticky {
+            position: sticky;
+            top: 0;
+            z-index: 800;
+            margin-bottom: 10px;
+            border-bottom: 1px solid #d4d4d5;
```
Comment (silverwind) [inline web_src/less/_repository.less:1575]: Need equivalent color in arc-green, probably `rgba(255,255,255,.1)`.

**Comment by silverwind** [comment]
Good call removing the sticky module, thanks.

**Review by silverwind — APPROVED** [review]

**Review by 6543 — APPROVED** [review]
now the glitch is gone

**Comment by techknowlogick** [comment]
ping lg-tm bot

**Pull request merged**